orbit/cmd/orbit/orbit.go

```diff
@@ -790,6 +789,10 @@ func main() {
 		return nil
 	}
 
+	if len(os.Args) == 2 && os.Args[1] == "--help" {
+		platform.RunUpdateQuirks()
```

Comment:
Maybe a more self explanatory name?

Reply:
Maybe something like  `RecoverPreviousInstallBlock` or similar?

Reply:
RunUpdateQuirks is meant to be a placeholder to put "software quirks" functions to fix stuff during update logic. We currently just have the symlink not present issue, but there might be others in the future. That's why I think it is better to keep this function generic. There is a more specific name for the function that actually implements the fix

Reply:
Yes, this is understood.
So maybe a name like: `PreInstallationQuirks`/`PreInstallationHooks`/...

Reply:
`PreUpdateQuirks` sounds good to me. (and add some documentation that such code should be reviewed with extra care and should only be used when auto-update has been broken.)

Reply:
Thanks. I've just added the documentation and renamed the function